pom.xml

```diff
@@ -113,7 +104,7 @@
                     the BSD 3-Clause license. For full license text, see the LICENSE.txt file in the
                     repository.
                 </bottom>
-                <additionalparam>-Xdoclint:none</additionalparam>
+                <doclint>none</doclint>
```

Comment:
The `maven-javadoc-plugin` 3.x requires `<doclint>` instead of `<additionalparam>`